tests/util/xds_local_test.go

```diff
@@ -0,0 +1,247 @@
+// Copyright 2018 Istio Authors
```

Comment:
Same - either in the v2 ( next to xds_test), or some other place - but likely not tests/util

Reply:
Same as above.